decorator.rb

```diff
@@ -0,0 +1,25 @@
+require_relative 'nameable'
+class Decorator
```

Comment:
- Hey, Your decorator class should inherit from the Nameable class. Please. kindly fix this.